lib/vty.c

```diff
@@ -2141,6 +2150,21 @@ static void vtysh_accept(struct event *thread)
 		close(sock);
 		return;
 	}
+	/* Setting the SND buffer size to 16MB to increase the buffer size
```

Comment:
please don't embed the current value in the comment

Reply:
Will do